docs/deployment.md

```diff
@@ -154,29 +153,30 @@ These buckets do not need any non-default permissions to function with Cumulus,
 
 **Note**: s3 bucket object names are global and must be unique across all accounts/locations/etc.
 
-#### Create IAM roles
+--------------
+## Configure and Deploy the IAM stack
 
-##### Add new deployment to `<daac>-deploy/iam/config.yml`
+##### Configure deployment with `<daac>-deploy/iam/config.yml`
 
 The `iam` configuration creates 4 [roles](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) and an [instance profile](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) used internally by the Cumulus stack.
 
-The various config fields are described below with a sample `config.yml` at the end.   All items in `<` `>` brackets are intended to be configured with user-set values:
+The various config fields are described below with a sample `config.yml` at the end.
 
 ------
 
-###### iam-deployment-name
+###### iam-deployment-name:
 
 The name (e.g. dev) of the the 'deployment' - this key tells kes which configuration set (in addition to the default values) to use when creating the cloud formation template[^4]
 
 ###### prefix:
 
-This value will prefix CloudFormation-created deployer resources and permissions.
+This value will prefix CloudFormation-created IAM resources and permissions.
```

Comment:
I think at some point, somehow, the `deployer` stack instructions got muddled into instructions for the `iam` and `app` stacks... The deploy stack for some reason doesn't even have it's own section so I'm just fixing the sections that exist. We might need to add back the deployer section but it's not clear to me that the permissions will be the same since we have NGAP 2.0 now anyways.

Reply:
Right, Alireza had suggested removing it because of NGAP 2.0